website/src/contribute/postcommits-policies.md

```diff
@@ -65,11 +65,34 @@ When a post-commit test fails, follow the provided steps for your situation.
 
 ### My change was rolled back due to a test failure {#pr-rolled-back}
 
-1.  Look at the JIRA issue to find the reason for the rollback.
-1.  Fix your code and re-run the post-commit tests.
-1.  Implement new pre-commit tests that will catch similar bugs before future
-    code is merged into the repository.
-1.  Open a new PR that contains your fix and the new pre-commit tests.
+After rollback there is time for deeper investigation. Start by looking at the
+JIRA issue to see the background information for the rollback. There are three
+common scenarios:
+
+1.  Your change contained a bug.
+2.  Your change exposed an existing bug.
+3.  Your change exposed a bad test (flaky, overspecified, etc).
+
+_These are all valid reasons for rollback. Maintaining clear signal is the
+highest priority._
+
+The high level steps are the same:
+
+1.  Create a fix and re-run the post-commit tests.
+2.  Implement new pre-commit tests that will catch similar failures
```

Comment:
nit: use `1.` everywhere and let the markdown processor convert to actual numbers.

Reply:
A primary purpose of markdown is to be readable in source form. Admittedly not critical for web page source, but very nice to have. Is this just to avoid touching adjacent lines, or is there a more compelling reason?

Reply:
(FWIW I think I've had think conversation before, and remember being vaguely convinced, but I just don't recall what the benefit was supposed to be)

Reply:
Yes, it reduces the maintenance effort when injecting a new item and having to re-number a long list.

This isn't a hard style-rule, and I see the argument for source readability. I don't feel strongly if you prefer to keep it as-is.